### PR TITLE
Paket

### PR DESCRIPTION
**Pull request by forki**
For testing purposes I tried to convert this project to Paket. http://fsprojects.github.io/Paket/

If  you are interested check it out.

### CODE REVIEW

**Comment by forki** [comment]
I updated everything to the latest Paket version.

File: FSharp.Charting.sln [hunk 1]
```diff
@@ -17,8 +23,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{5FF52F30-670D-4E8C-85FA-27F7A5D6C591}"
 	ProjectSection(SolutionItems) = preProject
 		.nuget\NuGet.Config = .nuget\NuGet.Config
```
Comment (tpetricek) [inline FSharp.Charting.sln]: Shouldn't this be also removed from the solution?

File: FSharp.Charting.Tests.sln [hunk 2]
```diff
@@ -19,8 +25,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{5FF52F30-670D-4E8C-85FA-27F7A5D6C591}"
 	ProjectSection(SolutionItems) = preProject
 		.nuget\NuGet.Config = .nuget\NuGet.Config
```
Comment (tpetricek) [inline FSharp.Charting.Tests.sln]: dtto

**Comment by forki** [comment]
done.

**Comment by forki** [comment]
tumblr_inline_my8brdkuoh1raprkq

**Pull request merged**